xamarin/Xamarin.Forms
# [Android] Fix RefreshView Content Margin issue

### Description of Change ###

Fixed layout issue on Android. Now the RefreshView content use margin…

<img width="449" alt="8986-droid" src="https://user-images.githubusercontent.com/6755973/76967193-5e99cc00-6927-11ea-9b68-ddb3e02b9ccc.png">

### Issues Resolved ### 

- fixes #8689 

### API Changes ###

 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Screenshots ### 

![issue8689](https://user-images.githubusercontent.com/6755973/76967600-f0093e00-6927-11ea-9e62-67f9c7f84449.gif)

### Testing Procedure ###
Launch Core Gallery on Android and navigate to the RefreshView samples. Select the new "RefreshView margins Gallery" sample. Use the slider to modify the RefreshView Content margin and verify that is working as expected.

### PR Checklist ###
